optaplanner-persistence/optaplanner-persistence-jpa/pom.xml

```diff
@@ -22,6 +22,22 @@
   </description>
   <url>https://www.optaplanner.org</url>
 
+  <properties>
+    <java.module.name>org.optaplanner.persistence.jpa</java.module.name>
+    <version.org.postgresql>42.2.12</version.org.postgresql>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.postgresql</groupId>
+        <artifactId>postgresql</artifactId>
+        <version>${version.org.postgresql}</version>
+        <scope>test</scope>
```

Comment:
By default, locally the test should run with h2.
Req: When I do "mvn clean install" for this module, it needs to run the tests and not require me to start up a database. So it must use an in memory database.

We can have a maven profile that adds postegresql and that is activated for this CI workflow.

Reply:
Everything you described is indeed the current behaviour of this PR. Doing a fresh `mvn clean install` here runs the tests in an in-memory h2 database (notice that the `h2` profile has `activeByDefault` set to true). This PR also adds a `postgresql` profile, which is activated when the user runs builds with `-Ppostgresql` like the GitHub workflow does (this requires a PostgreSQL server running in the background).

Reply:
Maybe I would just move this dependencyManagement to the top-level pom.xml, so that it's clear what versions we declare atop of the kie-parent pom.

Reply:
Good point @cuijulian - I missed that

Reply:
+1 to declare this in the top level pom. Ideally kie-parent in the 7.x series. Can be a separate PR pair.

Reply:
Sounds good. By 7.x do you mean `master` branch or `7.30.x` branch?